staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -61,6 +62,9 @@ const (
 	defaultBatchBufferSize = 1000        // Buffer up to 1000 events before blocking.
 	defaultBatchMaxSize    = 100         // Only send 100 events at a time.
 	defaultBatchMaxWait    = time.Minute // Send events at least once a minute.
+
+	defaultBatchMaxRoutine      = 10 // Send routine at 10 qps.
```

Comment:
defaultBatchThrottleQPS maybe?

Reply:
Should we limit this by default at all? /cc @soltysh

Reply:
I wanted to follow-up with a PR that adds a flag to control these values and enable/disable throttling

I'm not strongly against setting the flag to false by default, but IMO setting some limit by default is useful in case something goes wrong